lib/stsci/tools/asnutil.py

```diff
@@ -14,7 +14,9 @@
 from astropy.io import fits
 import numpy as N
 import os.path, time
+from distutils.version import LooseVersion
 
+ASTROPY_VER_GE13 = LooseVersion(astropy.__version__) >= LooseVersion('1.3')
 
 __version__ = '0.2(2015-06-23)'
```

Comment:
It is true that ``__version__`` could be updated but I kind of do not care for the following reasons:
1. This version is already in a bad format (version + version date!!!)
2. We do not use this anywhere
3. Our code uses different styles in different packages/modules => I think we must decide on a uniform way of versioning. Until that is done, I see no much reason to update version, especially for something as minor as the issue of clobber which will not affect functionality at all.